.jshintrc

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
@tombatossals moved the jshint settings to the .jshintrc file so editors can pick it up.